src/main/resources/com/google/api/codegen/java/standalone_sample.snip

```diff
@@ -284,7 +284,12 @@
 
 @private accessor(accessors)
   @join field : accessors on ""
-    .{@field}()
+    @switch field.kind
```

Comment:
nit: Above I would s/`concatenatedArgs`/`commaSeparatedFields`/g

In the other langs as well.

Reply:
Done